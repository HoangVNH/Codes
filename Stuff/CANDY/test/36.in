10
34
VXXXXXDDDDDVVVVVXVVVVVVVVVVDDDVVVV
21
DDDXXXXVDDDVVVXXXXXXX
37
DDDDDDDDDDDVVVDDDDDVVVVVXXXVVVVDDDDDD
37
XXXXXXXXXVVVVDDDDDDDXXXXVVVXXVVVVXXXX
37
XXXXXXXXXXDDDDXXXXVVVVXDDXXXXXXXXXXXX
25
VVVVVVVVDDDDDXXXXXDXXXXXX
47
DDDDDVVVXDDDDDDDDDVVDDDDDDDDDXXXXDDDDDDDDDDDXXV
42
DDDDDDDDXXXXXXXXXXXXVXXVDDDDDVVVXDDDDDXXXX
32
DDDDDDDDDXXXXXXXDDDDDVVVVVXXXXXV
28
VVVVVVVVVVVXXXXXXXXXXXXVVVDD
