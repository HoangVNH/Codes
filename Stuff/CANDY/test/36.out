32
20
36
35
36
24
46
41
30
26
